sys-libs/glibc/files/eblits/src_prepare.eblit

```diff
@@ -58,6 +58,8 @@ eblit-glibc-src_prepare() {
 	find . -type f '(' -size 0 -o -name "*.orig" ')' -delete
 	find . -name configure -exec touch {} +
 
+	eprefixify extra/locale/locale-gen
```

Comment:
Yes, the `eprefixify` function just runs `sed -i -e "s|@GENTOO_PORTAGE_EPREFIX@|${EPREFIX}|g"` on the given files.  That line was added upstream and was left there during the sync.